.github/workflows/acp-test.yaml

```diff
@@ -6,9 +6,13 @@ on:
     branches:
       - 'release/**'
     paths:
+      - 'charts/istio-authorizer/Chart.yaml'
```

Comment:
I thought that you have to adjust `branches` property in a `kube-acp-stack-test.yaml` file and `istio-authorizer-test.yaml`